lib/hydra/derivatives/processors/jpeg2k_image.rb

```diff
@@ -73,20 +73,18 @@ def layer_rates(layer_count, compression_numerator)
         rates.map(&:to_s).join(',')
       end
 
-      protected
```

Comment:
Thank you @val99erie for the clarification; Always hesitant to `public`-ize new methods, but this sounds like a necessary reason to open the public facing methods.